docs/development/releases.md

```diff
@@ -0,0 +1,33 @@
+# Synapse Release Cycle
+
+New releases of Synapse nominally follow a two week release cycle:
+
+* Day 0: Synapse N - 1 is released.
+* Day 7: Synapse N release candidate 1 is released.
+* Days 7 - 13: Synapse N release candidates 2+ are released, if bugs are found.
+* Day 14: Synapse N is released.
```

Comment:
I always find it hard to format these semi-maths in this kind of text in a way where they're readable as maths and not weird punctuation; some emphasis with italics or code blocks would possibly help:

```suggestion
* Day 0: Synapse `N - 1` is released.
* Day 7: Synapse `N` release candidate 1 is released.
* Days 7 - 13: Synapse `N` release candidates 2+ are released, if bugs are found.
* Day 14: Synapse `N` is released.
```
(alternatively an example can do better here, even if it means hardcoding some numbers)

Is it worth mentioning that we aim for Tuesdays?